### PR TITLE
Add some long time markers for faster doctesting

### PR DESCRIPTION
**Pull request by saraedum**
With these changes, the entire test suite runs again in under 10s for me. (on 48 cores :P)

### CODE REVIEW

File: flatsurf/geometry/minimal_cover.py [hunk 1]
```diff
@@ -34,10 +34,10 @@ class MinimalTranslationCover(Surface):
     The following is to test that unfolding is reasonably fast on the instances reported
     in https://github.com/flatsurf/sage-flatsurf/issues/47::
 
-        sage: T = polygons.triangle(2, 13, 26)
-        sage: S = similarity_surfaces.billiard(T, rational=True)
-        sage: alarm(5); S = S.minimal_cover("translation"); cancel_alarm()
-        sage: S
+        sage: T = polygons.triangle(2, 13, 26)  # long time, 3s in late 2021
```
Comment (saraedum) [inline flatsurf/geometry/minimal_cover.py:37]:
```suggestion
        sage: T = polygons.triangle(2, 13, 26)  # long time (3s)
```

File: flatsurf/geometry/minimal_cover.py [hunk 2]
```diff
@@ -34,10 +34,10 @@ class MinimalTranslationCover(Surface):
...
+        sage: S = similarity_surfaces.billiard(T, rational=True)  # long time
+        sage: alarm(5); S = S.minimal_cover("translation"); cancel_alarm()  # long time
+        sage: S  # long time
```
Comment (saraedum) [inline flatsurf/geometry/minimal_cover.py:40]:
```suggestion
        sage: S = similarity_surfaces.billiard(T, rational=True)  # long time (above)
        sage: alarm(5); S = S.minimal_cover("translation"); cancel_alarm()  # long time (above)
        sage: S  # long time (above)
```

File: flatsurf/geometry/minimal_cover.py [hunk 3]
```diff
@@ -110,10 +110,10 @@ class MinimalHalfTranslationCover(Surface):
...
     in https://github.com/flatsurf/sage-flatsurf/issues/47::
 
-        sage: T = polygons.triangle(2, 13, 26)
-        sage: S = similarity_surfaces.billiard(T, rational=True)
-        sage: alarm(5); S = S.minimal_cover("half-translation"); cancel_alarm()
-        sage: S
+        sage: T = polygons.triangle(2, 13, 26)  # long time, 3s in late 2021
+        sage: S = similarity_surfaces.billiard(T, rational=True)  # long time
+        sage: alarm(5); S = S.minimal_cover("half-translation"); cancel_alarm()  # long time
+        sage: S  # long time
```
Comment (saraedum) [inline flatsurf/geometry/minimal_cover.py:116]:
```suggestion
        sage: T = polygons.triangle(2, 13, 26)  # long time (3s)
        sage: S = similarity_surfaces.billiard(T, rational=True)  # long time (above)
        sage: alarm(5); S = S.minimal_cover("half-translation"); cancel_alarm()  # long time (above)
        sage: S  # long time (above)
```

**Comment by saraedum** [comment]
@videlec could you have a look here? Should any of these actually be fast? If so, we should create some follow-up issues.

File: flatsurf/geometry/gl2r_orbit_closure.py [hunk 4]
```diff
@@ -755,20 +755,20 @@ def lift(self, v):
 
             sage: T = polygons.triangle(3,4,13)
             sage: S = similarity_surfaces.billiard(T)
-            sage: S = S.minimal_cover("translation").erase_marked_points() # optional: pyflatsurf
-            sage: O = GL2ROrbitClosure(S)  # optional: pyflatsurf
-            sage: for d in O.decompositions(4, 20):  # optional: pyflatsurf
+            sage: S = S.minimal_cover("translation").erase_marked_points() # long time (3s), optional: pyflatsurf
+            sage: O = GL2ROrbitClosure(S)  # long time (above), optional: pyflatsurf
+            sage: for d in O.decompositions(4, 20):  # long time (2s), optional: pyflatsurf
```
Comment (videlec) [inline flatsurf/geometry/gl2r_orbit_closure.py:760]: Here I would say that it is suspicious dependsing on how many `d` are involved in this loop.

Comment (videlec) [inline flatsurf/geometry/gl2r_orbit_closure.py:760]: Here I would say that it is suspicious depending on how many `d` are involved in this loop before the `break`. I counted `3` which looks pretty low to me. Even on the second or third run I got something like 4 seconds for the whole block.

File: flatsurf/geometry/gl2r_orbit_closure.py [hunk 5]
```diff
@@ -1235,7 +1235,7 @@ def __reduce__(self):
             sage: S = similarity_surfaces.billiard(T)
             sage: S = S.minimal_cover(cover_type="translation")
             sage: O = GL2ROrbitClosure(S)  # optional: pyflatsurf
-            sage: loads(dumps(O)) == O
+            sage: loads(dumps(O)) == O  # long time (6s)
```
Comment (videlec) [inline flatsurf/geometry/gl2r_orbit_closure.py:1238]: Here I don't understand what could take so long. There is no subtle algorithm that is run to rebuild `O`.

File: flatsurf/geometry/gl2r_orbit_closure.py [hunk 6]
```diff
@@ -755,20 +755,20 @@ def lift(self, v):
...
-            sage: O2 = GL2ROrbitClosure(S2.surface())  # optional: pyflatsurf
-            sage: for d in O2.decompositions(4, 20):  # optional: pyflatsurf
+            sage: d1,d2,d3,d4 = [O.lift(b) for b in O.tangent_space_basis()]  # long time (above), optional: pyflatsurf
+            sage: dreal = d1/132 + d2/227 + d3/280 - d4/201  # long time (above), optional: pyflatsurf
+            sage: dimag = d1/141 - d2/233 + d4/230 + d4/250  # long time (above), optional: pyflatsurf
+            sage: d = [O.V2((x,y)).vector for x,y in zip(dreal,dimag)]  # long time (above), optional: pyflatsurf
+            sage: S2 = O._surface + d  # long time (6s), optional: pyflatsurf
+
+            sage: O2 = GL2ROrbitClosure(S2.surface())  # long time (above), optional: pyflatsurf
+            sage: for d in O2.decompositions(1, 20):  # long time (25s), optional: pyflatsurf
```
Comment (videlec) [inline flatsurf/geometry/gl2r_orbit_closure.py:771]: `25s` looks a bit excessive. Too many iterations? (I was not able to get the loop finished on my computer).

File: flatsurf/geometry/gl2r_orbit_closure.py [hunk 7]
```diff
@@ -755,20 +755,20 @@ def lift(self, v):
 
             sage: T = polygons.triangle(3,4,13)
             sage: S = similarity_surfaces.billiard(T)
-            sage: S = S.minimal_cover("translation").erase_marked_points() # optional: pyflatsurf
-            sage: O = GL2ROrbitClosure(S)  # optional: pyflatsurf
-            sage: for d in O.decompositions(4, 20):  # optional: pyflatsurf
+            sage: S = S.minimal_cover("translation").erase_marked_points() # long time (3s), optional: pyflatsurf
```
[the remark this answers is not in the document]
Comment (saraedum) [reply flatsurf/geometry/gl2r_orbit_closure.py:758]: This is now #122.

File: flatsurf/geometry/gl2r_orbit_closure.py (hunk 5 above)
Comment (saraedum) [reply flatsurf/geometry/gl2r_orbit_closure.py:1238]: I think that booting the pickle machinery for C++ types is taking a very long time.

Comment (saraedum) [reply flatsurf/geometry/gl2r_orbit_closure.py:1238]: This is now #123.

File: flatsurf/geometry/gl2r_orbit_closure.py (hunk 6 above)
Comment (saraedum) [reply flatsurf/geometry/gl2r_orbit_closure.py:771]: This is now #124.

File: flatsurf/geometry/gl2r_orbit_closure.py (hunk 4 above)
Comment (saraedum) [reply flatsurf/geometry/gl2r_orbit_closure.py:760]: This is now #124.

Comment (saraedum) [reply flatsurf/geometry/gl2r_orbit_closure.py:760]: This is now #124.

**Pull request merged**